model/project_ref.go

```diff
@@ -883,15 +888,49 @@ func (p *ProjectRef) UpdateAdminRoles(toAdd, toRemove []string) error {
 	return nil
 }
 
-func (p *ProjectRef) AddGitCloneToWorkstationCommands() {
-	if !p.WorkstationConfig.GitClone {
-		return
+func (p *ProjectRef) GitCloneCommand(directory string) *jasper.Command {
+	repoPath := fmt.Sprintf("git@github.com:%s/%s.git", p.Owner, p.Repo)
+
+	return jasper.NewCommand().AppendArgs("git", "clone", "-b", p.Branch, repoPath, directory)
```

Comment:
This command actually doesn't have Error/OutputSender or the grip.Info--can we add this logic?